src/modules/flight_mode_manager/tasks/Auto/FlightTaskAuto.cpp

```diff
@@ -159,11 +159,6 @@ bool FlightTaskAuto::update()
 	_checkEmergencyBraking();
 	Vector3f waypoints[] = {_prev_wp, _position_setpoint, _next_wp};
 
-	if (isTargetModified()) {
```

Comment:
@MaEtUgR could you double check that this check is indeed no longer needed?

Reply:
👀 
The check was introduced in:
https://github.com/PX4/PX4-Autopilot/commit/33702be3e4e81d237c9975f71e8f6969e13e2bb1#diff-3b9266c5fffe78c73d047b6ee8cf3e9ef9d9eacd7cc74db660a718a4e957b754R205
adapted here:
https://github.com/PX4/PX4-Autopilot/commit/fdd1d6d2446daa7c9b905fbd7bbd9cebd0f7555a#diff-3b9266c5fffe78c73d047b6ee8cf3e9ef9d9eacd7cc74db660a718a4e957b754R210
and seems to be avoidance only since:
https://github.com/PX4/PX4-Autopilot/commit/fed234de81d33c6002f1713c266238da78fba762#diff-3fa0514afc596aefc8754a909f5ed57ce5c80429e559679c5e98e5ca7118b226R114-R117

So yes I'm also assuming it's only used for avoidance.